faq/views.py

```diff
@@ -0,0 +1,12 @@
+from django.views import generic
+
+from .models import Question
+
+
+class FAQListView(generic.ListView):
+    template_name = 'faq/list.html'
+    model = Question
+
+    def get_queryset(self):
+        queryset = super(FAQListView, self).get_queryset()
+        return queryset.filter(published=True).order_by('category')
```

Comment:
I am pretty sure you can replace line 8 to line 12 by using `queryset`directly (https://github.com/django/django/blob/master/django/views/generic/list.py#L26) : 
```python
queryset = Question.objects.filter(published=True).order_by('category')
```